cli/command/node/demote.go

```diff
@@ -18,6 +18,9 @@ func newDemoteCommand(dockerCli command.Cli) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return runDemote(cmd.Context(), dockerCli, args)
 		},
+		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			return completeNodeNames(dockerCli)(cmd, args, toComplete)
```

Comment:
```suggestion
		ValidArgsFunction: completeNodeNames(dockerCli)
```

nit

Reply:
Doh! Good call; wouldn't surprise me if we did the same in other places (sometimes the closures trip me up for sure!)